launchcontainer/launchcontainer.py

```diff
@@ -132,11 +132,11 @@ class LaunchContainerXBlock(XBlock):
         help=("Enables students to reset/delete their container and start over")
     )
 
-    support_email = String(
-        display_name='Tech support email',
-        default=getattr(settings, "TECH_SUPPORT_EMAIL", ""),
+    support_url = String(
+        display_name='Tech support url',
+        default=getattr(settings, "TECH_SUPPORT_URL", ""),
         scope=Scope.content,
-        help=("Email address of tech support for AVL labs."),
+        help=("Link to tech support for AVL labs."),
```

Comment:
Step 1) Remove the default support email, so it works only if the customer sets it explicitly.

```py
    support_email = String(
        display_name='Tech support email',
        default=None,
        scope=Scope.content,
        help=("Email address of tech support for AVL labs."),
    )
```